src/camd/gxccd.cpp

```diff
@@ -441,14 +466,23 @@ int GXCCD::startExposure ()
 	{
 		gxccd_get_last_error (camera, gx_err, sizeof (gx_err));
 		logStream (MESSAGE_ERROR) << "GXCCD::startExposure error calling gxccd_set_binning " << gx_err << sendLog;
+		if (ret == -1)
+			reinitCamera ();
 		return -1;
 	}
 
-	ret = gxccd_start_exposure (camera, getExposure (), getExpType () == 0, getUsedX (), getUsedY (), getUsedWidth (), getUsedHeight ());
+	// The GXCCD library expects pre-binned region coordinates and size
+	ret = gxccd_start_exposure (camera, getExposure (), getExpType () == 0,
+								(int) (getUsedX () / binningHorizontal ()),
+								(int) (getUsedY () / binningVertical ()),
```

Comment:
getUsedWidthBinned() etc. will not fix it?

Reply:
There are no binned analogs for getUsedX, so I decided not to add them but just be more verbose in both cases. Should I introduce binned getUsed[XY] versions?